.github/workflows.src/nightly.tpl.yml

```diff
@@ -15,6 +15,13 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
+    - name: Install Rust toolchain
```

Comment:
Technically, we don't need the up-to-date rust toolchain for Linux, because we run the build in a container, which already has it.

Reply:
That's right, I'll remove that then for clarity. The edgedb/edgedb equivalent isn't installing the Rust toolchain on Linux either.